### ISSUE TITLE
page not found

### ISSUE THREAD

**Issue by akshayamadhuri** [knative-sandbox/eventing-kafka-broker#3214]
**Describe the bug**
[Eventing Kafka Channel Docs](https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/docs/channel/README.md) the links in this file are not found

 **Expected behavior**

 **To Reproduce**

 **Knative release version**

 **Additional context**
Add any other context about the problem here such as proposed priority

**Comment by Cali0707** [comment]
Nice catch, thanks!
/triage-accepted

**Comment by akshayamadhuri** [comment]
assign/

**Comment by akshayamadhuri** [comment]
@Cali0707  i will wait for @pierDipi comment

**Comment by pierDipi** [comment]
Hi @akshayamadhuri, thanks for reporting, in this case, I'd reword the text to use the past and link to previous branches where those other channel implementation existed while also clarifying that now the other implementations have been removed
